2005-10-13  Ulrich Drepper  <drepper@redhat.com>
	
	[BZ #1373]
	* argp/argp.h: Remove __NTH for __argp_usage inline function.
	
===================================================================
RCS file: /cvs/glibc/libc/argp/argp.h,v
retrieving revision 1.30
retrieving revision 1.31
diff -u -r1.30 -r1.31
--- libc/argp/argp.h	2004/09/07 22:23:45	1.30
+++ libc/argp/argp.h	2005/10/14 05:53:47	1.31
@@ -561,7 +561,7 @@
 # endif
 
 ARGP_EI void
-__NTH (__argp_usage (__const struct argp_state *__state))
+__argp_usage (__const struct argp_state *__state)
 {
   __argp_state_help (__state, stderr, ARGP_HELP_STD_USAGE);
 }
